packages/story-editor/src/app/helpCenter/provider.js

```diff
@@ -213,6 +214,10 @@ const createBooleanMapFromKey = (key) =>
 
 function HelpCenterProvider({ children }) {
   const { additionalTips = {} } = useConfig();
+  const {
+    actions: { updateCurrentUser: _updateCurrentUser },
+  } = useAPI();
+  const canUpdateCurrentUser = Boolean(_updateCurrentUser);
```

Comment:
Could probably move this to `CurrentUserProvider`

Reply:
You mean pass `canUpdateCurrentUser` as state from `CurrentUserProvider` ? We could but, that doesn't look very intuitive because `canUpdateCurrentUser` sounds more like a capability which it is not.